rewrite-java/src/main/java/org/openrewrite/java/style/IntelliJ.java

```diff
@@ -87,4 +88,8 @@ public static SpacesStyle spaces() {
     public static WrappingAndBracesStyle wrappingAndBraces() {
         return new WrappingAndBracesStyle();
     }
+
+    public static UnnecessaryParenthesesStyle unnecessaryParentheses() {
+        return new UnnecessaryParenthesesStyle(true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true);
```

Comment:
Haha this file is an argument for Java to support named parameters if I've ever seen one! I have no suggested change or improvement, I just think it's funny to see a sequence of `true, true, true, ...` trailing off the edge of the page.

Reply:
lol. That's 22 `true`'s. Giving _wide screen monitors_ the marketing justification they need 😉